unattended_scripts/install_functions/opendistro/elasticsearch.sh

```diff
@@ -182,7 +182,7 @@ configureElasticsearch() {
     eval "rm /etc/elasticsearch/certs/client-certificates.readme /etc/elasticsearch/certs/elasticsearch_elasticsearch_config_snippet.yml -f ${debug}"
     eval "/usr/share/elasticsearch/bin/elasticsearch-plugin remove opendistro-performance-analyzer ${debug}"
 
-    initializeElastic nip
+    initializeElastic
```

Comment:
`Elastic` is the company name. The component is `Elasticsearch`. Please change all occurrences.